test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/InProcTestDicomWebServer.cs

```diff
@@ -86,6 +86,10 @@ public InProcTestDicomWebServer(Type startupType, TestServerFeatureSettingType[]
                 }
 
                 config.AddEnvironmentVariables();
+                if (string.Equals(existingConfig["DicomServer:Features:EnableExternalStore"], bool.TrueString, StringComparison.OrdinalIgnoreCase))
```

Comment:
We don't need to worry about this now, but there is probably a better way than what we're doing

Reply:
Yeah, and when it comes to propagating flags/settings across both the host and the tests themselves. We can probably leverage XUnit DI community NuGet package, but that's part of a larger refactor